doc/nestml_language/nestml_language_concepts.rst

```diff
@@ -1121,6 +1121,23 @@ Integrating the ODEs needs to be triggered explicitly inside the ``update`` bloc
 
 The ``integrate_odes()`` function numerically integrates the differential equations defined in the ``equations`` block. Integrating the ODEs from one timestep to the next has to be explicitly carried out in the model by calling the ``integrate_odes()`` function. If no parameters are given, all ODEs in the model are integrated. Integration can be limited to a given set of ODEs by giving their left-hand side state variables as parameters to the function, for example ``integrate_odes(V_m, I_ahp)`` if ODEs exist for the variables ``V_m`` and ``I_ahp``. In this example, these variables are integrated simultaneously (as one single system of equations). This is different from calling ``integrate_odes(V_m)`` and then ``integrate_odes(I_ahp)`` in that the second call would use the already-updated values from the first call. Variables not included in the call to ``integrate_odes()`` are assumed to remain constant (both inside the numeric solver stepping function as well as from before to after the call).
 
+In case of higher-order ODEs of the form ``F(x'', x', x) = 0``, the solution to ``x(t)`` can be integrated by just providing the variable ``x`` to the ``integrate_odes`` function. For example,
```

Comment:
```suggestion
In case of higher-order ODEs of the form ``F(x'', x', x) = 0``, the solution ``x(t)`` can be integrated by just providing the variable ``x`` to the ``integrate_odes`` function. For example,
```

Reply:
"...the solution ``x(t)`` is obtained by..."

It's the ODE that is "integrated", not the solution.